submissions/damilolaSalami/domManipulation/signUpform.js

```diff
@@ -0,0 +1,39 @@
+const submitform = document.getElementById("validateForm"),
+    firstname = document.getElementById("firstname"),
+    lastname = document.getElementById("lastname"),
+    phonenumber = document.getElementById("phoneNumber"),
+    email = document.getElementById("email"),
+    password = document.getElementById("password"),
+    confirmpassword= document.getElementById("confirmpassword");
+
+const nameRegex =  /^[a-zA-Z ]+$/,
+    pnumberRegex = /^[+]*[0-9]+$/,
+    emailRegex = /^\w+([-]?\w+)*@\w+([-]?\w+)*(\.\w{2,6})+$/,
+    passwordRegex = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[._$,/;:+=~`'"<>^#?!@$%^&*-]).{6,}$/;
+
+function formValidation (field, Regex, error_one, error_two, event) {
```

Comment:
I think `Regex` is a reserved word, kindly change it

Reply:
I think `Regex` is a reserved word, kindly change the name of that parameter to something else